src/Sidekick.Common/Game/Items/Category.cs

```diff
@@ -21,4 +21,6 @@ public enum Category
     MemoryLine = 16,
     Affliction = 17,
```

Comment:
Still exists?

Reply:
Technically no, but I didn't want to remove categories. Our data provider does not use the numbers anymore to determine the category. Maybe we can clean these later